app/pb_kits/playbook/pb_progress_pills/_progress_pills.jsx

```diff
@@ -0,0 +1,27 @@
+/* @flow */
+/*eslint-disable react/no-multi-comp, flowtype/space-before-type-colon */
+
+import React from 'react'
+
+type ProgressPillsProps = {
+  active?: String,
+  steps?: String,
+}
+
+const printSteps = (steps, active) => {
+  let items = []
+
+  for(let step = 1; step <= steps; step++) {
+    items.push(<span className={step <= active ? "active" : "inactive"}></span>)
+  }
+
+  return items
+}
+
+const ProgressPills = ({ active = 0, steps = 0 }: ProgressPillsProps) => (
+  <div className="pb_progress_pills_kit">
+    {printSteps(steps, active)}
```

Comment:
How can I do this loop in a better way? I don't like it, but the JSX does not accept the `{for(...)}` syntax.
I looked for some like `[1..steps].each ...` but I was not able to find it.

Reply:
@rafbgarcia do you have any ideas on how to make it looks better?

Reply:
Does this work?

```js
const Step = ({ active, step }) => (
  <>
    <span className={classnames({ active: step <= active, inactive:  step > active }></span>)
    <span> </span>
  </>
)

{steps.map((step) => <Step step={step} active={active} />)}

Reply:
Ah sorry I made a confusion on the `steps` variable type.

Reply:
I found this `Array.apply(null, { length: 10 }).map(Number.call, Number)`, which returns an array from 0 to `length`

```js
Array.apply(null, { length: steps }).map(Number.call, Number).map((step) => (
  <Step 
      step={step} 
      active={active} 
  />
)
```